action.yml

```diff
@@ -11,29 +11,24 @@ inputs:
     description: 'The path to scan. This option is mutually exclusive with "image".'
     required: false
   debug:
-    description: 'Set this to any value to enable verbose debug output'
+    description: "Set this to any value to enable verbose debug output"
     required: false
-    default: 'false'
+    default: "false"
   fail-build:
-    description: 'Set to false to avoid failing based on severity-cutoff. Default is to fail when severity-cutoff is reached (or surpassed)'
-    required: false
-    default: 'true'
-  grype-version:
-    description: 'Optionally, specify the Grype version (e.g. 0.1.0) to use instead of the default version'
+    description: "Set to false to avoid failing based on severity-cutoff. Default is to fail when severity-cutoff is reached (or surpassed)"
     required: false
+    default: "true"
   acs-report-enable:
-    description: 'Optionally, enable feature that causes a result.sarif report to be generated after successful action execution.  This report is compatible with GitHub Automated Code Scanning (ACS), as the artifact to upload for display as a Code Scanning Alert report.'
+    description: "Optionally, disable the feature that causes a result.sarif report to be generated after successful action execution.  This report is compatible with GitHub Automated Code Scanning (ACS), as the artifact to upload for display as a Code Scanning Alert report."
```

Comment:
Ditto: I've updated the text here to also to remove the reference to `results.sarif`, as this might change and the end user should use the output parameter `sarif`.